pages/apis/rest_api/clusters.md

```diff
@@ -344,8 +377,72 @@ curl "https://api.buildkite.com/v2/organizations/{org.slug}/clusters/{cluster.id
 }
 ```
 
-Required scope: `read_clusters`
+Optional [request body properties](/docs/api#request-body-properties):
+
+<table class="responsive-table">
+<tbody>
+  <tr>
+    <th><code>note</code></th>
+    <td>
+      Note explaining why the queue is paused. The note will display on the queue page and any affected builds.
+      <br><em>Example:</em> <code>"Paused while we investigate a security issue"</code>
+    </td>
+  </tr>
+</tbody>
+</table>
+
+Required scope: `write_clusters`
 
 Success response: `200 OK`
 
+Error responses:
+
+<table class="responsive-table">
+<tbody>
+  <tr><th><code>422 Unprocessable Entity</code></th><td><code>{ "message": "Cluster queue is already paused" }</code></td></tr>
+</tbody>
+</table>
+
+### Resume a paused queue
 
+```bash
+curl -X POST "https://api.buildkite.com/v2/organizations/{org.slug}/clusters/{cluster.id}/queues/{id}/resume_dispatch" \
+  -H "Content-Type: application/json"
+```
+
+```json
+{
+  "id": "01885682-55a7-44f5-84f3-0402fb452e66",
+  "graphql_id": "Q2x1c3Rlci0tLTQyZjFhN2RhLTgxMmQtNDQzMC05M2Q4LTFjYzdjMzNhNmJjZg==",
+  "key": "Default Queue",
+  "description": "The default queue for this cluster",
+  "url": "http://api.buildkite.com/v2/organizations/test/clusters/42f1a7da-812d-4430-93d8-1cc7c33a6bcf/queues/01885682-55a7-44f5-84f3-0402fb452e66",
+  "web_url": "http://buildkite.com/organizations/test/clusters/42f1a7da-812d-4430-93d8-1cc7c33a6bcf/queues/01885682-55a7-44f5-84f3-0402fb452e66",
+  "cluster_url": "http://api.buildkite.com/v2/organizations/test/clusters/#{cluster.id}",
```

Comment:
I think it would be good to replace `#{cluster.id}` with a realistic value to be consistent with the other fields. There are a few other cases in the responses to update, too.

(This [came up on a related PR](https://github.com/buildkite/docs/pull/2155/files#r1211106972) and sounds like this was a copy/paste error)

Reply:
Ah sorry I think this is an old diff because I was based on that branch, I'll rebase now.

Reply:
Done, thanks @mbelton-buildkite